src/runtime/manifest-parser.pegjs

```diff
@@ -1837,6 +1837,34 @@ MultiplicativeExpression
     return leftExpr;
   }
 
+FunctionArguments
+  = arg: PaxelExpressionWithRefinement rest:(multiLineSpace? ',' multiLineSpace? PaxelExpressionWithRefinement)*
+  {
+    return [arg, ...rest.map(item => item[3])];
+  }
+
+FunctionCall
+  = fn: (fieldName '(' FunctionArguments? ')' / 'creationTimestamp')
```

Comment:
fieldName is confusing as function name, maybe:

FunctionCall
  = fn: (name:lowerIdent '(' FunctionArguments? ')' / 'creationTimestamp')

Reply:
lowerIdent seems to break query args ('?'), so I left it was fieldName for now.